sdk/servicebus/service-bus/src/dataTransformer.ts

```diff
@@ -124,7 +125,11 @@ export const defaultDataTransformer = {
 export function isRheaAmqpSection(
   possibleSection: any | RheaAmqpSection
 ): possibleSection is RheaAmqpSection {
-  return possibleSection != null && typeof possibleSection.typecode === "number";
+  return (
+    possibleSection != null &&
+    typeof possibleSection.typecode === "number" &&
+    isDefined(possibleSection.content)
```

Comment:
Is it possible for the `content` to be null/undefined? What if I had sent a message with no body, but all other properties set?

Reply:
Yeah, I'd leave out the content check . We could possibly improve this by making sure typecode is one of the 3 values we accept, otherwise I think it's a "good enough" check for something used internally.

Reply:
Just checked.. it can be undefined/null.

If set undefined and sent, it comes back as null.

Reply:
Added the check to make sure the received typecode is of the expected ones.